unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/SceneController.cs

```diff
@@ -123,6 +125,8 @@ private void Update()
                 sceneSortDirty = false;
                 SortScenesByDistance();
             }
+
+            performanceMetricsController?.Update();
```

Comment:
Would like to see this on top of the `Update` call. My thinking is: how does this behave, if we miss a frame?

Reply:
`Update` is called all the frames. If a frame takes 100ms, this'd be called at the start of the next frame. 

This is just the result of collapsing `Update` calls after the `PerformanceMetricsController` lost its `MonoBehaviour` status. There should be not any meaningful difference in flow from the previous commits.